modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_item_selector_preview.scss

```diff
@@ -113,6 +113,6 @@ $infoPanelSidebarDDColor: $secondary-l1;
 	}
 
 	.image-editor {
-		height: calc(100% - 3.5rem);
+		height: calc(100% - 115px);
```

Comment:
I personally prefer to continue using `rem` units.

`height: calc(100% - 7.19rem);`

Reply:
In this case is the same since not header nor toolbar height are defined neither in `rem` or `px` but dynamically calculated by the browser depending on the content. That's why to me it feels easier to understand why this calc (100% - header height - toolbar height) is that way, rather than a relative unit inside an already dynamic `calc`. But as I said the resulto is the same so I don't have a very strong opinion about this.

Reply:
I'm suggesting this because we have others calc with the same pattern in this file, but maybe you are right, and feels easier to understand in this way.